lib/core/userinfo/components/user-info.component.stories.ts

```diff
@@ -0,0 +1,202 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+    Meta,
+    moduleMetadata,
+    Story,
+    componentWrapperDecorator
+} from '@storybook/angular';
+import { CoreStoryModule } from '../../testing/core.story.module';
+import { UserInfoComponent } from './user-info.component';
+import { UserInfoModule } from '../userinfo.module';
+import { PeopleContentService } from './../../services/people-content.service';
+import { BpmUserService } from './../../services/bpm-user.service';
+import { IdentityUserService } from './../../services/identity-user.service';
+import { AuthenticationService } from './../../services/authentication.service';
+import { AuthenticationServiceMock } from './mocks/authentication.service.mock';
+import {
+    BpmUserServiceStub,
+    IdentityUserServiceStub,
+    PeopleContentServiceStub
+} from './mocks/user.service.mock';
+
+export default {
+    component: UserInfoComponent,
+    title: 'Core/User Info/User Info',
+    decorators: [
+        moduleMetadata({
+            imports: [CoreStoryModule, UserInfoModule],
+            providers: [
+                {
+                    provide: PeopleContentService,
+                    useClass: PeopleContentServiceStub
+                },
+                {
+                    provide: BpmUserService,
+                    useClass: BpmUserServiceStub
+                },
+                {
+                    provide: IdentityUserService,
+                    useClass: IdentityUserServiceStub
+                },
+                {
+                    provide: AuthenticationService,
+                    useClass: AuthenticationServiceMock
+                }
+            ]
+        }),
+        componentWrapperDecorator(
+            (story) =>
+                `<div style="height: 100vh; display: flex; align-items: center; justify-content: center;">
+                    ${story}
+                </div>`
+        )
```

Comment:
We decided not to use componentWrapperDecorators and similar solutions, unless necessary. Can you check if it can be done with `template`?

Reply:
I think that this is the best solution in this case - I need only to center a div to show how some properties are working. `Template` requires from me binding this properties and here I can use only `${story}`.

Reply:
👍